app/views/menus/edit.html.haml

```diff
@@ -0,0 +1,7 @@
+%h1= "#{t :edit_menu}:"
+
+= render partial: 'meals/courses', collection: @facade.courses, locals: {facade: @facade}
+
+= render partial: 'meals/form', locals: { menu: @facade.menu, meal: @facade.new_meal }
```

Comment:
No need to use `partial` & `locals` unless you're rendering a collection.
`render 'meals/form', menu: @facade.menu, meal: @facade.new_meal`